shopfloor_packing_info/tests/test_checkout_scan_line.py

```diff
@@ -26,6 +26,10 @@ def test_scan_line_package_ok_packing_info_filled_info(self):
         self._fill_stock_for_moves(move2, in_package=True)
         picking.action_assign()
         move_line = move1.move_line_ids
+        related_lines = picking.move_line_ids - move_line
         self._test_scan_line_ok(
-            move_line.package_id.name, move_line, packing_info=packing_info_text
+            move_line.package_id.name,
+            move_line,
+            related_lines,
+            packing_info=packing_info_text,
```

Comment:
@JuMiSanAr I think you should also revert this change as you reverted the change on the package scan.

You can also rebase and squash